database/migrations/2024_10_03_150604_create_delayed_jobs_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('delayed_jobs', function (Blueprint $table) {
+            $table->id();
+            $table->uuid('uuid')->unique();
+            $table->enum('status', ['pending', 'failed', 'succeeded']);
+            $table->integer('statusCode')->nullable();
+            $table->string('payload');
```

Comment:
This should be a `json` and needs to be nullable.

Reply:
done